aries-site/src/layouts/content/ContentSection.js

```diff
@@ -4,8 +4,6 @@ import { Box } from 'grommet';
 export const ContentSection = ({ ...rest }) => (
   <Box
     as="section"
-    align="start"
-    gap="medium"
```

Comment:
Is this `gap` being replaced by the addition of the `margin.bottom` in `SubsectionText.js`?

Reply:
Yes, it's being swapped out there. Because now we don't have as much control to wrap sections in `Subsection` or `ContentSection` to control spacing and so text controls its own spacing more.